webapp/controllers/api/DataProvider.js

```diff
@@ -43,10 +42,17 @@ module.exports = function(app) {
                 active: dataProviderReceived.active || false
               };
 
+              if (uriObject.protocol == 'FTP'){
```

Comment:
Prefer to use identity comparison (`===`) than equality (`==`) as much as possible to avoid JS "gotchas". It avoids the JS interpreter to perform cast operations turning out unexpected behavior.